packages/runtime/src/components/core/Stack.tsx

```diff
@@ -0,0 +1,111 @@
+import { implementRuntimeComponent } from '../../utils/buildKit';
+import React, { ReactNode } from 'react';
+import { css, CSSObject } from '@emotion/css';
+import { Static, Type } from '@sinclair/typebox';
+import { StringUnion } from '../../utils/stringUnion';
+
+export const StackPropertySchema = Type.Object({
+  align: StringUnion(['start', 'end', 'center', 'baseline'], {
+    title: 'Align Items',
+    category: 'Layout',
+  }),
+  direction: StringUnion(['vertical', 'horizontal'], {
+    title: 'Direction',
+    category: 'Layout',
+  }),
+  justify: StringUnion(
+    ['flex-start', 'flex-end', 'center', 'space-between', 'space-around'],
+    {
+      title: 'Justify Content',
+      category: 'Layout',
+    }
+  ),
+  spacing: Type.Number({
+    title: 'Spacing',
+    category: 'Layout',
+  }),
+  wrap: Type.Boolean({
+    title: 'Wrap',
+    category: 'Layout',
+    description: 'Auto wrap line, when horizontal effective',
+  }),
+});
+
+const StateSchema = Type.Object({});
+
+export type StackProps = Static<typeof StackPropertySchema> & {
+  cssStyle?: string;
+};
+
+const Stack = React.forwardRef<HTMLDivElement, StackProps & { children: ReactNode }>(
+  (
+    { cssStyle, align, direction = 'horizontal', spacing = 12, children, wrap, justify },
+    ref
+  ) => {
+    const [horizontalSize, verticalSize] = React.useMemo(
+      () => (Array.isArray(spacing) ? spacing : [spacing, spacing]),
+      [spacing]
+    );
+
+    const style: CSSObject = {
+      alignItems: align,
+      justifyContent: justify,
+      flexDirection: direction === 'vertical' ? 'column' : 'row',
+      display: 'inline-flex',
+      columnGap: horizontalSize,
+      rowGap: verticalSize,
+    };
+
+    if (wrap) {
+      style.flexWrap = 'wrap';
```

Comment:
Why not just adding this line the declaration of `style`?

Reply:
There was other logic that has since been removed, so I'll change it